### PR TITLE
add more lesson

### PR DESCRIPTION
**Pull request by tarigancana**

### CODE REVIEW

File: src/in/2018-01/03/07.md [hunk 1]
```diff
@@ -1,7 +1,18 @@
----
-title:  Pendalaman
-date:   19/01/2018
----
-
-### <center>Kami sedang mengerjakan pelajaran ini</center>
-<center>Silahkan kembali lagi nanti</center>
\ No newline at end of file
+---
+title:  Pendalaman
```
Comment (VitalikL) [inline src/in/2018-01/03/07.md]: @tarigancana I think title is missing here

File: src/in/2018-01/05/07.md [hunk 2]
```diff
@@ -1,7 +1,18 @@
----
-title:  Daily Lesson
-date:   02/02/2018
----
-
-### <center>Kami sedang mengerjakan pelajaran ini</center>
-<center>Silahkan kembali lagi nanti</center>
\ No newline at end of file
+---
+title:  Pendalaman
```
Comment (VitalikL) [inline src/in/2018-01/05/07.md]: @tarigancana And here

File: src/in/2018-01/03/07.md (hunk 1 above)
Comment (tarigancana) [reply src/in/2018-01/03/07.md]: that's on line 2 is the title

File: src/in/2018-01/05/07.md (hunk 2 above)
Comment (tarigancana) [reply src/in/2018-01/05/07.md]: line 2 is the title :)

**Pull request merged**